app/views/admin/assets/show.html.erb

```diff
@@ -405,5 +405,6 @@
     <% else %>
       <p class="mt-3 mb-3">No OCR for this asset.</p>
     <% end %>
+    <%= render"upload_hocr_and_pdf", asset: @asset unless @asset.suppress_ocr %>
```

Comment:
So, the ones we WANT to manually upload tesseract products for.. have `suppress_ocr` set, because we don't want the nightly process trying to OCR and getting a guaranteed error!

So that's the actual use case.... I think this conditional will mean the manual upload won't be available for the actual use case we have?

Reply:
Oh, you're absolutely right! Will remove that.

Reply:
And then I was thinking before... should the suppress_ocr bit be automatically _unset_ after succesfully uploading both files? (See also below about uploading both files together)

Reply:
Yeah - of course we can try doing that.

Reply:
OH and then though... how do we get the work re-indexed after a succesful assignment of manual teseract products, so it will show up in our site search?  

Should we just re-index it after succesful assignment?  (Maybe in a bg job?). If someone is uploading manual prdoucts for _multiple_ assets in a work, it'll get reindexed after each one, so multiple times unnecessarily... but that's prob not a big deal and is not an expected use case anyway.

Reply:
Yup - good call. We already reindex the work on any change to any member asset's OCR status; not a problem.